Answers.md

```diff
@@ -1,11 +1,11 @@
 # Answers
 
-1.  What is React JS and what problems does it try and solve?
+1.  What is React JS and what problems does it try and solve? React is a library used to write large scale application. With software changing so fast it allows us to write software and deploy our changes rapidly enough to keep up.
 
-1.  What does it mean to _think_ in react?
+1.  What does it mean to _think_ in react? When you use data to visualize and drive your views.

```

Comment:
Right on, here is a blog that talks about thinking in react in detail. Mastering this concept will make your dev time be more productive, meaning a better developer.
https://reactjs.org/docs/thinking-in-react.html

Reply:
I you add my account as a contributor, I can also merge your PR.